src/board_controller/openbci/ganglion.cpp

```diff
@@ -592,7 +641,22 @@ int Ganglion::call_open ()
         }
 
         safe_logger (spdlog::level::info, "search for {}", params.mac_address.c_str ());
-        res = func (const_cast<char *> (params.mac_address.c_str ()));
+
+        struct ConnectionParameters
```

Comment:
I dont like strcut definition like this in cpp code and it seems like its duplicated between ganglion.cpp and bglib, can be moved to a common header I think. Also, compiler may align it randomly somewhere(its unlikely)